domain/domain.go

```diff
@@ -398,6 +407,48 @@ func (do *Domain) PrivilegeHandle() *privileges.Handle {
 	return do.privHandle
 }
 
+func (do *Domain) loadTableStats() error {
+	ver, err := do.store.CurrentVersion()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	snapshot, err := do.store.GetSnapshot(kv.NewVersion(ver.Ver))
+	if err != nil {
+		return errors.Trace(err)
+	}
+	m := meta.NewSnapshotMeta(snapshot)
+	err = do.statsHandle.Update(m, do.InfoSchema())
+	return errors.Trace(err)
+}
+
+// LoadTableStatsLoop creates a goroutine loads stats info in a loop, it
+// should be called only once in BootstrapSession.
+func (do *Domain) LoadTableStatsLoop(ctx context.Context) error {
+	do.statsHandle = statscache.NewHandle(ctx)
+	err := do.loadTableStats()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	lease := do.DDL().GetLease()
+	if lease > 0 {
+		go func(do *Domain) {
+			ticker := time.NewTicker(lease)
```

Comment:
When a user execute analyze, we must notify others as soon as possible. I think a lease is ok.